lib/workers/repository/update/pr/body/controls.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { getControls } from './controls';
+
+describe('workers/repository/update/pr/body/controls', () => {
+  it('calls getControls', () => {
```

Comment:
This is needed as `getControls()` is not covered anywhere, but being mocked instead.

![image](https://user-images.githubusercontent.com/97394622/201544692-4be4ca68-0a1a-498a-a855-e0ec449428bd.png)